cmake/Config.cmake

```diff
@@ -28,7 +47,27 @@ find_package (Boost 1.53.0
         thread
         system
         unit_test_framework
-        python)
+        ${_BOOSTPYLIB})
+
+# if _BOOSTPYLIB was not found for non-windows platforms,
+# try searching for built-from-source defaults
+if (NOT WIN32 AND NOT ${Boost_${_BOOSTPYLIB}_FOUND})
+    if (${PYTHON_VERSION_MAJOR} STREQUAL 2)
+        set (_BOOSTPYLIB "python")
+    else()
+        set (_BOOSTPYLIB "python${PYTHON_VERSION_MAJOR}")
+    endif()
+    find_package (Boost 1.53.0
+        OPTIONAL_COMPONENTS
+            date_time
+            thread
+            system
+            unit_test_framework
+            ${_BOOSTPYLIB})
+endif()
+# remap boost python vars
+set(Boost_PYTHON_FOUND ${Boost_${_BOOSTPYLIB}_FOUND})
```

Comment:
This breaks add_python_subdirectory macro defined in PythonTest.cmake and as the result the Python tests/examples are not included in the build. It should be:

set(Boost_PYTHON_FOUND Boost_${_BOOSTPYLIB}_FOUND)